docs/docs-content/getting-started/aws/scale-secure-cluster.md

```diff
@@ -0,0 +1,505 @@
+pack. Click on the **hellouniverse 1.1.3** layer. The pack manifest editor appears.
+
+Click on **Presets** on the right-hand side. You can learn more about the pack presets on the pack README, which is
+available in the Palette UI. Select the **Enable Hello Universe API** preset. The pack manifest changes accordingly.
+
+![Screenshot of pack presets](/getting-started/aws/getting-started_scale-secure-cluster_pack-presets.webp)
+
+The pack requires two values to be replaced for the authorization token and for the database password when using this
+preset. Replace these values with your own base64 encoded values. The
+[_hello-universe_](https://github.com/spectrocloud/hello-universe?tab=readme-ov-file#single-load-balancer) repository
+provides a token that you can use.
+
+Click on **Confirm Updates**. The manifest editor closes. Then, click on **Save Changes** to save your updates.
+
+## Deploy a Cluster
+
+Navigate to the left **Main Menu** and select **Clusters**. Click on **Create Cluster**.
+
+Palette will prompt you to select the type of cluster. Select **AWS IaaS** and click on **Start AWS IaaS
+Configuration**.
+
+Continue with the rest of the cluster deployment flow using the cluster profile you created in the
+[Import a Cluster Profile](#import-a-cluster-profile) section, named **aws-profile**. Refer to the
+[Deploy a Cluster](./deploy-k8s-cluster.md#deploy-a-cluster) tutorial for additional guidance or if you need a refresher
+of the Palette deployment flow.
+
+### Verify the Application
+
+Navigate to the left **Main Menu** and select **Clusters**.
+
+Select your cluster to view its **Overview** tab.
+
+When the application is deployed and ready for network traffic, Palette exposes the service URL in the **Services**
+field. Click on the URL for port **:8080** to access the Hello Universe application.
+
+![Cluster details page with service URL highlighted](/getting-started/aws/getting-started_scale-secure-cluster_service_url.webp)
+
+## Upgrade Kubernetes Versions
+
+Regularly upgrading your Kubernetes version is an important part of maintaining a good security posture. New versions
+may contain important patches to security vulnerabilities and bugs that could affect the integrity and availability of
+your clusters.
+
+Palette supports three minor Kubernetes versions at any given time. We support the current release and the three
+previous minor version releases, also known as N-3. For example, if the current release is 1.29, we support 1.28, 1.27,
+and 1.26.
+
+:::warning
+
+Once you upgrade your cluster to a new Kubernetes version, you will not be able to downgrade.
+
+:::
+
+We recommend using cluster profile versions to safely upgrade any layer of your cluster profile and maintain the
+security of your clusters. Expand the following section to learn how to create a new cluster profile version with a
+Kubernetes upgrade.
+
+<details>
+
+<summary>Upgrade Kubernetes using Cluster Profile Versions</summary>
```

Comment:
I liked the details component, but Im not 100% sure we need it as this is an important part of the tutorial

Reply:
I chose to hide it because we cover cluster profile versions in detail in the Cluster Profile Updates tutorial, which is earlier in Getting Started.